### PR TITLE
#168978974 Build API endpoint for sharing articles with database

### PR DESCRIPTION
**Pull request by hirwaf**
#### What does this PR do?
Having 'create article' API endpoint working
#### Description of Task to be completed?
Having this endpoint working
`/api/v2/articles`
#### How should this be manually tested?
After cloning the repo cd into it and run `npm run migrate & npm start`
- Using Postman test `/api/v2/articles` endpoint with this header
key: **Content-Type** 
value: **application/json**
method: **POST**
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/168978974
#### Screenshots (if appropriate)
Screenshot_20191008_233610_articles_saved
#### Questions:
N/A

### CODE REVIEW

File: server/database/migrations/create_employees_table.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class CreateEmployeesTable {
       + 'firstName VARCHAR (60) NOT NULL,'
       + 'lastName VARCHAR (60) NOT NULL,'
       + 'email VARCHAR (60) NOT NULL,'
-      + 'password VARCHAR (60) NOT NULL,'
+      + 'password VARCHAR (255) NOT NULL,'
```
Comment (ekumamait) [inline server/database/migrations/create_employees_table.js:15]: was there an issue with keeping the password at (60)

**Review by ekumamait — COMMENTED** [review]
I would like us to take some time today and review one of the files, let me know when you are ready

File: server/database/migrations/create_employees_table.js (hunk 1 above)
Comment (hirwaf) [reply server/database/migrations/create_employees_table.js:15]: Yes, because password encryption 
bcrypt generate a string with more than 60.

Comment (ekumamait) [reply server/database/migrations/create_employees_table.js:15]: Alright makes sense

**Pull request merged**